core/mondoo-linux-workstation-security.mql.yaml

```diff
@@ -208,7 +208,21 @@ queries:
           permissions.other_writeable == false
           permissions.other_executable == false
          }
-       }
+      }
+
+      if ( file("/boot/loader/loader.conf").exists ) {
```

Comment:
You'll want to add this file to the remediation steps

Reply:
@tas50 thanks for the hint. fixed